Dockerfile

```diff
@@ -16,6 +16,8 @@ COPY --from=builder /go/src/github.com/openshift/builder/openshift-builder /usr/
 COPY imagecontent/policy.json /etc/containers/
 COPY imagecontent/registries.conf /etc/containers/
 COPY imagecontent/storage.conf /etc/containers/
+RUN mkdir /var/cache/blobs
```

Comment:
There's some discussion about changing this name to "var/cache/blobs/metadata" assuming this is the metadata cache.  Please sync with @runcom and @mrunalp

Reply:
related to https://github.com/openshift/origin/pull/21613#discussion_r245744542

Reply:
`/var/cache/blobs/meta` I've used that in 21613 suggested by @mrunalp

Reply:
nm, this dir is apparently for something else.